rviz_rendering/include/rviz_rendering/logging.hpp

```diff
@@ -1,31 +1,35 @@
-/*
- * Copyright (c) 2017, Open Source Robotics Foundation, Inc.
- * All rights reserved.
- *
- * Redistribution and use in source and binary forms, with or without
- * modification, are permitted provided that the following conditions are met:
- *
- *     * Redistributions of source code must retain the above copyright
- *       notice, this list of conditions and the following disclaimer.
- *     * Redistributions in binary form must reproduce the above copyright
- *       notice, this list of conditions and the following disclaimer in the
- *       documentation and/or other materials provided with the distribution.
- *     * Neither the name of the Willow Garage, Inc. nor the names of its
- *       contributors may be used to endorse or promote products derived from
- *       this software without specific prior written permission.
- *
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
- * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
- * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
- * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
- * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
- * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
- * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
- * POSSIBILITY OF SUCH DAMAGE.
- */
+//
+// Copyright (c) 2017, Open Source Robotics Foundation, Inc.
+// All rights reserved.
+//
+// Software License Agreement (BSD License 2.0)
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions
+// are met:
+//
+//  * Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+//  * Redistributions in binary form must reproduce the above
+//    copyright notice, this list of conditions and the following
+//    disclaimer in the documentation and/or other materials provided
+//    with the distribution.
+//  * Neither the name of the Open Source Robotics Foundation, Inc. nor the names of its
```

Comment:
I don't think this is the right change because Willow Garage's name should still not be used (nothing has changed in that respect).

The updated 3-clause bsd template uses a more general statement here:

> Neither the name of the copyright holder nor the names of its contributors may be used to endorse
-- https://choosealicense.com/licenses/bsd-3-clause/

We should probably update to that instead.

Also there is a pull request I am looking into that should allow us to use the block comments rather than `//` comments, letting us have smaller diffs:

https://github.com/ament/ament_lint/pull/82

So if you want to merge this, for now I would suggest disabling the copyright linter and wait for that pr to get merged before trying to enable it again. Otherwise I am going to wait to merge this pr until we have that sorted out because I really don't want to have a diff like this in all the files we port.

Reply:
I agree - it would be much better to have ament_lint copyright check be able to work with block comments. If this happens soon, I'll be inclined to wait for it. Otherwise, I can also disable it if you want.
Regarding the change, I wasn't sure (that's why it's in a separate commit) - it just seemed logical because the file has no connection whatsoever to Willow Garage. 

The template I chose was the one recognized by ament_lint - see here: https://github.com/ament/ament_lint/tree/master/ament_copyright/ament_copyright/template
- the weird part I only saw now is that the license template is equivalent to the bsd-3-clause you mention, while the license header isn't - and ament_copyright checks for the header.